src/Symfony/Component/Notifier/Message/SentMessage.php

```diff
@@ -45,4 +46,14 @@ public function getMessageId(): ?string
     {
         return $this->messageId;
     }
+
+    public function getDebugData(): ?array
+    {
+        return $this->debugData;
+    }
+
+    public function setDebugData(array $data): void
```

Comment:
this should instead be a new constructor argument

Reply:
I chose the setter method initially because some transporters may work differently and not have http-response or extra-data to include?

So, it would make sense to make it an optional same as `setMessageId()`?

Not sure but just clarifying